tests/unit/ApiService.spec.ts

```diff
@@ -100,4 +100,15 @@ describe('function: createPeginTx', () => {
     expect(result.outputs[2].op_return_data).to.be.equal('test1');
   });
 
+  it('obtain api version, return promise', async () => {
+    setEnvironment(true, true);
+
+    try {
+      const result = await ApiService.getApiVersion();
+      expect(result.version).not.to.be.equal('0');
+    } catch (e)
+    {
+      expect(e).to.be.a('error', 'Invalid data when obtain API Version');
```

Comment:
why are you expecting to have an error?

Reply:
- changed to catch when API route is unavailable.